db/migrate/20181005180705_remove_unused_columns_from_feedback_messages.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveUnusedColumnsFromFeedbackMessages < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :feedback_messages, :last_reviewed_at
+    remove_column :feedback_messages, :offender_email_sent?
+    remove_column :feedback_messages, :reporter_email_sent?
+    remove_column :feedback_messages, :victim_email_sent?
+    rename_column :feedback_messages, :victim_id, :affected_id
```

Comment:
`rename_column` could have been a standalone migration since it doesn't fall into the category of the migration name "Remove Unused Columns"